vars/runFunctionalTest.groovy

```diff
@@ -0,0 +1,112 @@
+    export SCT_CONFIG_FILES=${test_config}
+    export SCT_COLLECT_LOGS=false
+
+    if [[ -n "${params.aws_region ? params.aws_region : ''}" ]] ; then
+        export SCT_REGION_NAME=${aws_region}
+    fi
+
+    if [[ -n "${params.new_version ? params.new_version : ''}" ]] ; then
+        export SCT_NEW_VERSION="${params.new_version}"
+    fi
+
+    if [[ -n "${params.k8s_scylla_operator_docker_image ? params.k8s_scylla_operator_docker_image : ''}" ]] ; then
+        export SCT_K8S_SCYLLA_OPERATOR_DOCKER_IMAGE=${params.k8s_scylla_operator_docker_image}
+    fi
+    if [[ -n "${params.k8s_scylla_operator_helm_repo ? params.k8s_scylla_operator_helm_repo : ''}" ]] ; then
+        export SCT_K8S_SCYLLA_OPERATOR_HELM_REPO=${params.k8s_scylla_operator_helm_repo}
+    fi
+    if [[ -n "${params.k8s_scylla_operator_chart_version ? params.k8s_scylla_operator_chart_version : ''}" ]] ; then
+        export SCT_K8S_SCYLLA_OPERATOR_CHART_VERSION=${params.k8s_scylla_operator_chart_version}
+    fi
+
+    if [[ -n "${params.scylla_mgmt_agent_version ? params.scylla_mgmt_agent_version : ''}" ]] ; then
+        export SCT_SCYLLA_MGMT_AGENT_VERSION=${params.scylla_mgmt_agent_version}
+    fi
+
+    if [[ -n "${params.scylla_mgmt_agent_repo ? params.scylla_mgmt_agent_repo : ''}" ]] ; then
+        export SCT_SCYLLA_MGMT_AGENT_REPO=${params.scylla_mgmt_agent_repo}
+    fi
+
+    if [[ -n "${params.scylla_ami_id ? params.scylla_ami_id : ''}" ]] ; then
+        export SCT_AMI_ID_DB_SCYLLA="${params.scylla_ami_id}"
+    elif [[ -n "${params.gce_image_db ? params.gce_image_db : ''}" ]] ; then
+        export SCT_GCE_IMAGE_DB="${params.gce_image_db}"
+    elif [[ -n "${params.scylla_version ? params.scylla_version : ''}" ]] ; then
+        export SCT_SCYLLA_VERSION="${params.scylla_version}"
+    elif [[ -n "${params.scylla_repo ? params.scylla_repo : ''}" ]] ; then
+        export SCT_SCYLLA_REPO="${params.scylla_repo}"
+    elif [[ "${params.backend ? params.backend : ''}" == *"k8s"* ]] ; then
+        echo "Kubernetes backend can have empty scylla version. It will be taken from defaults of the scylla helm chart"
+    else
+        echo "need to choose one of SCT_AMI_ID_DB_SCYLLA | SCT_SCYLLA_VERSION | SCT_SCYLLA_REPO | SCT_GCE_IMAGE_DB"
+        exit 1
+    fi
+
+    if [[ -n "${params.oracle_scylla_version ? params.oracle_scylla_version : ''}" ]] ; then
+        export SCT_ORACLE_SCYLLA_VERSION="${params.oracle_scylla_version}"
+    fi
+
+    if [[ -n "${params.gemini_seed ? params.genini_seed : ''}" ]] ; then
+        export SCT_GEMINI_SEED="${params.gemini_seed}"
+    fi
+
+    export SCT_POST_BEHAVIOR_DB_NODES="${params.post_behavior_db_nodes}"
+    export SCT_POST_BEHAVIOR_LOADER_NODES="${params.post_behavior_loader_nodes}"
+    export SCT_POST_BEHAVIOR_MONITOR_NODES="${params.post_behavior_monitor_nodes}"
+
+    if [[ -n "${params.provision_type ? params.provision_type : ''}" ]] ; then
+        export SCT_INSTANCE_PROVISION="${params.provision_type}"
+    fi
```

Comment:
The same here. Do we really need it?

Reply:
This is not specific for k8s, it is general functional suite